docs/manual.txt

```diff
@@ -421,6 +421,14 @@ other static site generators. The currently supported metadata formats are:
 
 You can add arbitrary meta fields in any format.
 
+When you create new posts, by default the metadata will be created as reST style comments.
+If you prefer a different format, you can set the ``METADATA_FORMAT`` to one of these values:
+
+* ``"Nikola"``: reSt comments wrapped in a comment if needed (default)
```

Comment:
I think Chris means that you typed `reSt` instead of `reST` (note the last `T`), not that you should rename the format :)